src/containers/Subscription.css

```diff
@@ -56,6 +56,10 @@
   .checkbox input:checked ~ .checkmark {
     background-color: #606060;
   }
+  .grayed-out{
+    background-color: #a0a0a0 !important;
```

Comment:
I haven't been doing any frontend work for a while so I'm not very sure, but I think generally we'd like to avoid using `!important` as it override all the other ones. Using id or class should be sufficient. Ignore me if that's not the case ;)